website/src/pages/podcast.tsx

```diff
@@ -2,6 +2,51 @@ import * as React from 'react'
+<line stroke="currentColor" x1="17" y1="16.5" x2="17" y2="23.5" id="line4646" fill="currentColor" strokeWidth="4" strokeLinecap="round" />
+<line stroke="currentColor" x1="17" y1="29.5" x2="17" y2="31.5" id="line4648" fill="currentColor" strokeWidth="4" strokeLinecap="round" />
+<line stroke="currentColor" x1="31" y1="24.5" x2="31" y2="31.5" id="line4650" fill="currentColor" strokeWidth="4" strokeLinecap="round" />
+<line stroke="currentColor" x1="31" y1="18.5" x2="31" y2="16.5" id="line4652" fill="currentColor" strokeWidth="4" strokeLinecap="round" />
+<line stroke="currentColor" x1="38" y1="23" x2="38" y2="25" id="line4654" fill="currentColor" strokeWidth="4" strokeLinecap="round" />
+<line stroke="currentColor" x1="10" y1="23" x2="10" y2="25" id="line4656" fill="currentColor" strokeWidth="4" strokeLinecap="round" />
+</svg>
+export const spotifyIcon = <svg aria-hidden="true" data-icon="spotify" viewBox="0 0 496 512" className="podcast__icon" xmlns="http://www.w3.org/2000/svg">
+<title>Spotify icon</title>
+<path fill="currentColor" d="M248 8C111.1 8 0 119.1 0 256s111.1 248 248 248 248-111.1 248-248S384.9 8 248 8zm100.7 364.9c-4.2 0-6.8-1.3-10.7-3.6-62.4-37.6-135-39.2-206.7-24.5-3.9 1-9 2.6-11.9 2.6-9.7 0-15.8-7.7-15.8-15.8 0-10.3 6.1-15.2 13.6-16.8 81.9-18.1 165.6-16.5 237 26.2 6.1 3.9 9.7 7.4 9.7 16.5s-7.1 15.4-15.2 15.4zm26.9-65.6c-5.2 0-8.7-2.3-12.3-4.2-62.5-37-155.7-51.9-238.6-29.4-4.8 1.3-7.4 2.6-11.9 2.6-10.7 0-19.4-8.7-19.4-19.4s5.2-17.8 15.5-20.7c27.8-7.8 56.2-13.6 97.8-13.6 64.9 0 127.6 16.1 177 45.5 8.1 4.8 11.3 11 11.3 19.7-.1 10.8-8.5 19.5-19.4 19.5zm31-76.2c-5.2 0-8.4-1.3-12.9-3.9-71.2-42.5-198.5-52.7-280.9-29.7-3.6 1-8.1 2.6-12.9 2.6-13.2 0-23.3-10.3-23.3-23.6 0-13.6 8.4-21.3 17.4-23.9 35.2-10.3 74.6-15.2 117.5-15.2 73 0 149.5 15.2 205.4 47.8 7.8 4.5 12.9 10.7 12.9 22.6 0 13.6-11 23.3-23.2 23.3z"></path>
+</svg>
+export const rssIcon = <svg aria-hidden="true" data-icon="rss" viewBox="0 0 448 512" className="podcast__icon" xmlns="http://www.w3.org/2000/svg">
+<title>RSS icon</title>
+<path fill="currentColor" d="M128.081 415.959c0 35.369-28.672 64.041-64.041 64.041S0 451.328 0 415.959s28.672-64.041 64.041-64.041 64.04 28.673 64.04 64.041zm175.66 47.25c-8.354-154.6-132.185-278.587-286.95-286.95C7.656 175.765 0 183.105 0 192.253v48.069c0 8.415 6.49 15.472 14.887 16.018 111.832 7.284 201.473 96.702 208.772 208.772.547 8.397 7.604 14.887 16.018 14.887h48.069c9.149.001 16.489-7.655 15.995-16.79zm144.249.288C439.596 229.677 251.465 40.445 16.503 32.01 7.473 31.686 0 38.981 0 48.016v48.068c0 8.625 6.835 15.645 15.453 15.999 191.179 7.839 344.627 161.316 352.465 352.465.353 8.618 7.373 15.453 15.999 15.453h48.068c9.034-.001 16.329-7.474 16.005-16.504z"></path>
+</svg>
+
+interface HTMLParts {
+    guestsHTML: string
+    audioHTML: string
+    summaryHTML?: string
+    transcriptHTML?: string
+    showNotesHTML?: string
+}
+
+export function getHTMLParts(html: string): HTMLParts {
+    const doc = document.createElement('html')
```

Comment:
```suggestion
    const doc = document.createElement('html')
```
I tried to review locally and received an error for 'document'.  Should this use:
public componentDidMount() ?

Reply:
I fixed this issue, so it should work locally now. Still need to fix the outstanding build errors.